language/move-prover/spec-lang/src/env.rs

```diff
@@ -2553,12 +2553,14 @@ impl<'env> FunctionEnv<'env> {
     }
 
     /// Returns the function name excluding the address and the module name
+    //TODO: return slice rather than owned strings.
+    #[allow(clippy::rc_buffer)]
```

Comment:
@dariorussi or whomever in move land, some of these lints were a little to large for me to want to roll them in to this update, so marked with allow.    At some point the debt will need addressed.    Seems RC/ARC should be used with slices rather than vecs (and str rather than String).